### DISCUSSION TITLE
Installation problem of90

### DISCUSSION THREAD

**Discussion by igoripcp** [fppimenta/rheoTool#87] [Q&A]
Hello,

When installing the version of rheoTool for of90, an error occurs: "cannot open output file /opt/openfoam9/platforms/linux64GccDPInt32Opt/lib/librtTwoPhaseChangeModels.so: Permission denied"

I checked, the librtTwoPhaseChangeModels.so library, which is being tried to be included at compile time, is not in the list of openfoam9 libraries.

Please help solve this problem?

The computer has a clean ubuntu 20.04 and the only version of OF-9.0 installed.

**Comment by fppimenta** [comment]
sudo chmod -R 777 opt/openfoam9

cd rheoTool && ./Allwclean && ./Allwmake

**Reply by fppimenta** [reply]
Also fixed by commit https://github.com/fppimenta/rheoTool/commit/3ee9995f6635429c31f6a99beed8334941d57354

**Reply by igoripcp** [reply]
I installed the fixed version, everything works fine. Thanks
